integrations/gen_doc_collector_page.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+"""
+This script reads the integrations/integrations.js file and generates the list of data collection integrations inside collectors/COLLECTORS.md, with proper links that Learn can replace into Learn links.
+"""
+
+# Open integrations/integrations.js and extract the dictionaries
+with open('integrations/integrations.js') as dataFile:
+    data = dataFile.read()
+
+    categories_str = data.split("export const categories = ")[1].split("export const integrations = ")[0]
+    integrations_str = data.split("export const categories = ")[1].split("export const integrations = ")[1]
+
+    categories = json.loads(categories_str)
+    integrations = json.loads(integrations_str)
+
+cat_dict = {}
+data_col_cat = {}
+
+
+def recursive(categories):
+    for category in categories:
+        data_col_cat[category['id']] = category['name']
+        if category['children']:
+            recursive(category['children'])
+
+
+recursive(categories[1]['children'])
+
+
+def construct_dict(array, integration):
+    for element in array:
+        try:
+            cat_dict[element].append(integration)
+        except KeyError:
+            cat_dict[element] = []
+            cat_dict[element].append(integration)
```

Comment:
```suggestion
        if element in cat_dict:
            cat_dict[element].append(integration)
        else:
            cat_dict[element] = [integration]
```

This more concretely describes the logic here, and should also be faster in the case of `element` not existing in `cat_dict` yet.

Reply:
nit-pick: I think it should be

```python
if element not in cat_dict:
    cat_dict[element] = list()
cat_dict[element].append(integration)
```

or just simply use [defaultdict](https://docs.python.org/3/library/collections.html#defaultdict-examples)

Reply:
This is even more efficient :smile: I will update and push again